docs/output.md

```diff
@@ -108,44 +138,93 @@ The [bigWig](https://genome.ucsc.edu/goldenpath/help/bigWig.html) format is in a
 
 The creation of these files can be bypassed by setting the parameters `--skip_bigwig`/`--skip_bigbed`.
 
-*Output directories*:
+## Transcript Reconstruction and Quantification
 
-* `<ALIGNER>/bigwig/`  
-  Per-sample `*.bigWig` files.
-* `<ALIGNER>/bigbed/`  
-  Per-sample `*.bigBed` files.
+<details markdown="1">
+<summary>Output files</summary>
 
-## MultiQC
+If bambu is used:
+
+* `bambu/`  
+  * `extended_annotations.gtf` - a gtf file that contains both annotated and novel transcripts  
+  * `counts_gene.txt` - gene expression estimates
+  * `counts_transcript.txt` - transcript expression estimates
+
+If StringTie2 is used:
+
+* `stringtie2/`
+  * `*.bam`
+     Per-sample coordinate sorted alignment files in [`*.bam`](https://samtools.github.io/hts-specs/SAMv1.pdf) format.
+  * `*.stringtie.gtf`
+     Per-sample annotations for novel transcripts obtained in *StringTie2*.
+  * `stringtie.merged.gtf`
+     Extended annotation that combines provided gtf with gtf files from each sample via *StringTie2 Merge*.
+  * `counts_gene.txt` - gene expression estimates calculated by featureCounts.
+  * `counts_gene.txt.summary` - featureCounts gene level log file.
+  * `counts_transcript.txt` - transcript expression estimates calculated by featureCounts.
+  * `counts_transcript.txt.summary` - featureCounts transcript level log file.
+
+</details>
 
 *Documentation*:  
-[MultiQC](https://multiqc.info/docs/)
+[bambu](https://github.com/GoekeLab/bambu), [StringTie2](https://ccb.jhu.edu/software/stringtie/), [featureCounts](http://bioinf.wehi.edu.au/featureCounts/)
```

Comment:
```suggestion
[bambu](https://bioconductor.org/packages/release/bioc/html/bambu.html), [StringTie2](https://ccb.jhu.edu/software/stringtie/), [featureCounts](http://bioinf.wehi.edu.au/featureCounts/)
```

Reply:
Fixed in https://github.com/nf-core/nanoseq/pull/89/commits/8be94a19fc87aaf34f1c2bc6572f6432624301b1